decidim-core/app/helpers/decidim/organization_helper.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Decidim
+  # View helpers related to the organization.
+
+  module OrganizationHelper
+    include Decidim::TranslatableAttributes
+
+    # Renders a view with the customizable CSS variables in two flavours:
+    # 1. as a hexadecimal valid CSS color (ie: #ff0000)
+    # 2. as a disassembled RGB components (ie: 255 0 0)
+    #
+    # Example:
+    #
+    # --primary: #ff0000;
+    # --primary-rgb: 255 0 0
+    #
+    # Hexadecimal variables can be used as a normal CSS color:
+    #
+    # color: var(--primary)
+    #
+    # While the disassembled variant can be used where you need to manipulate
+    # the color somehow (ie: adding a background transparency):
+    #
+    # background-color: rgba(var(--primary-rgb), 0.5)
+    def organization_colors
+      css = current_organization.colors.each.map { |k, v| "--#{k}: #{v};--#{k}-rgb: #{v[1..2].hex} #{v[3..4].hex} #{v[5..6].hex};" }.join
+      render partial: "layouts/decidim/organization_colors", locals: { css: }
+    end
+
+    def organization_description_label
+      @organization_description_label ||= if empty_organization_description?
+                                            t("decidim.pages.home.footer_sub_hero.footer_sub_hero_body_html")
+                                          else
+                                            decidim_sanitize_admin(translated_attribute(current_organization.description))
+                                          end
+    end
+
+    def organization_name(organization = current_organization)
+      translated_attribute(organization.name, organization)
+    end
+
+    def current_organization_name
+      organization_name(current_organization)
+    end
```

Comment:
I see that sometimes we use `current_organization_name`, but other times we use `organization_name(@organization)` and we also do `organization_name` (without parameters). I'd prefer to have it consistent always, and do the same call always so it's consistent, and it means the same. I'm leaning to `current_organization_name` as it's easy to remember.

Would that be possible? Or there's something that I'm missing (i.e. `current_organization` is not always available). If that's the case, for instance for mailers I'd prefer to define the `current_organization_name` method in the `ApplicationMailer` and try this kind of strategies (this can go to a subsequent PR if you feel that this can be too big to tackle in this PR 😄 )

Reply:
I have went and read all the files from above, and i would say they are correctly implemented. 
- All the erb Views that are being handled by controllers we are using `current_organization_name`
- All the mailers do implement `organization_name(....)`
- All the places where we need to display the organization name starting from a param is using `organization_name(....)`
- Specs are using `translated(organization_name)`

Starting from your comment, I have changed a few places where this was fit to do. 
Being a 150+ Files PR, i would not start changing the mailers so that we can use `current_organization_name`.